src/chapters/setup-intro-data-types/studio.rst

```diff
@@ -1,2 +1,68 @@
 Studio: Data Types
 ===================
+
+Get cosy with Java syntax by writing a console program that calculates the 
+area of a circle based on input from the user.
+
+Creating your class
+-------------------
+
+Since you’re still new to Java and IntelliJ, we’ll provide some extra
+direction the first few times we go to write code.
+
+Create a new package named ``org.launchcode.java.studios`` by
+right-clicking (or ctrl-clicking for some Mac users) on the ``src`` directory 
+in ``java-web-dev-exercises`` and selecting *New > Package*. Be sure to enter 
+the full name, or your package won’t be created in the correct location.
+
+Create your program/class in the ``java-web-dev-exercises`` project within the
+package ``org.launchcode.java.studios`` by right-clicking/ctrl-clicking on the
+``studios`` package/folder and selecting *New > Java Class*. Enter the
+name ``Area``. Select the option to add the file to Git when the window appears.
+
+Your task
+---------
+
+Write a program/class ``Area`` that prompts the user for a number, and
```

Comment:
The earlier text paired program with class ("program/class"), and further use of hashes just gets distracting. We should settle on one term to use for the remainder of the instructions.  I'd suggest "class".

Also, rather than prompting for "a number", we should probably be specific and use, "...the radius of a circle, and then calculate its area and print the result."